docker-test-framework/etc/irods-ext/metalnx.properties

```diff
@@ -73,12 +69,7 @@ metalnx.enable.tickets=false
 metalnx.enable.upload.rules=false
 # download limit to be set as 100MB
 metalnx.download.limit=6000
-# Global switch to allow view of a 'basic data and metadata' info view when the user does not have permission to the underlying file.
-# this allows a path to be resolved via the rods proxy admin account so that a 'no access' page is displayed with the ability to request access.
-# This covers cases where an external search has generated a link to a file the user has no existing permission to see
-# show dashboard (off by default due to performance issues)
```

Comment:
might be a duplicate comment... not sure where the earlier one went...

you've removed the comment... but not the dashboard enable flag itself?

i know that removing the dashboard is #213 ... is this related or part of that work instead?

Reply:
My interpretation was that that comment was more about the `access.proxy` but I now see the last line is for the dashboard.  What I'll do is restore the last line for now until I remove the dashboard.